### PR TITLE
Fix SeatGuru and Delta

### PR DESCRIPTION
**Pull request by lensovet**
* SeatGuru has changed their parameter for the departure airport
* Delta now requires a cabin class parameter (see https://www.flyertalk.com/forum/29315172-post519.html)

### CODE REVIEW

**Comment by lensovet** [comment]
As a side note, I didn't update the minified version because I wasn't sure how to do it.

**Comment by lensovet** [comment]
tagging @adamhwang to review and merge :)

File: ita-matrix-powertools.user.js [hunk 1]
```diff
@@ -3208,7 +3208,7 @@ function bindSeatguru(){
             printNotification("Error: Could not find target in bindSeatguru");
             return false;
           } else {
-            var url='http://www.seatguru.com/findseatmap/findseatmap.php?carrier='+currentItin['itin'][i]['seg'][j]['carrier']+'&flightno='+currentItin['itin'][i]['seg'][j]['fnr']+'&date='+('0'+currentItin['itin'][i]['seg'][j]['dep']['month']).slice(-2)+'%2F'+('0'+currentItin['itin'][i]['seg'][j]['dep']['day']).slice(-2)+'%2F'+currentItin['itin'][i]['seg'][j]['dep']['year']+'&from_loc='+currentItin['itin'][i]['seg'][j]['orig'];
+            var url='http://www.seatguru.com/findseatmap/findseatmap.php?carrier='+currentItin['itin'][i]['seg'][j]['carrier']+'&flightno='+currentItin['itin'][i]['seg'][j]['fnr']+'&date='+('0'+currentItin['itin'][i]['seg'][j]['dep']['month']).slice(-2)+'%2F'+('0'+currentItin['itin'][i]['seg'][j]['dep']['day']).slice(-2)+'%2F'+currentItin['itin'][i]['seg'][j]['dep']['year']+'&to=&from='+currentItin['itin'][i]['seg'][j]['orig'];
```
Comment (adamhwang) [inline ita-matrix-powertools.user.js]: @lensovet is `to` supposed to be empty?

Comment (lensovet) [reply ita-matrix-powertools.user.js]: yeah it works fine with that. i guess we could improve this to also include the destination airport but i figured i'd keep my changes to a minimum.

**Comment by adamhwang** [comment]
Thanks @lensovet!

**Comment by lensovet** [comment]
Ugh, I wasn't looking super carefully. It looks like the site now hardcodes the parameters into a cookie so SeatGuru is completely broken. However their AJAX endpoint works with these arguments…what's really needed here is for the link to show the SG result table inline, and then you can click that to get the seat map…

**Pull request merged**